catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -564,6 +568,8 @@ public void destroy() {
     config.put("isHistoricalSearchDisabled", !historicalSearchEnabled);
     config.put("isArchiveSearchDisabled", !archiveSearchEnabled);
     config.put("isMetacardPreviewDisabled", !metacardPreviewEnabled);
+    config.put("isCustomTextNotationEnabled", customTextNotationEnabled);
+    config.put("customTextNotationAttribute", customTextNotationAttribute);
```

Comment:
:question: Could we just have one config for this? I think `isCustomTextNotationEnabled` logic could be replaced by `customTextNotationAttribute === ''` or something similar

Reply:
Yes, that sounds good to me.

Reply:
Actually I think we do need the two configs because we want to have the ability for users to explicitly turn on/off this feature in the Admin Console. The default value is "title", so it would never be blank.

Reply:
I think this should be more of a user controllable runtime toggle